### ISSUE TITLE
Consider setting value to default value if it exists

### ISSUE THREAD

**Issue by nphmuller** [mrin9/RapiDoc#53]
Say I have to following request body:
``` json
"requestBody": {
  "content": {
    "application/x-www-form-urlencoded": {
      "schema": {
        "type": "object",
        "properties": {
          "username": {
            "type": "string"
          },
          "password": {
            "type": "string",
            "format": "password"
          },
          "grant_type": {
            "type": "string",
            "default": "password"
          }
        }
      }
    }
  }
}
```

I'd expect the UI to have to value of the `grant_type` parameter to be set to `password` by default. Instead it's empty.

I tried this via: https://mrin9.github.io/RapiDoc/examples/example2.html
If the example uses the latest commit it should be on: `ec5eb03bef7efe7845845a75b7e27eeecce0c36b`.
If it uses the latest release it should be on: `5.2.2`.
(I haven't found a way to determine the version from the example page itself)

**Comment by mrin9** [comment]
>Using default to specify a sample value. This is not intended use of default and can lead to unexpected behavior in some Swagger tools. Use the example or examples keyword for this purpose instead. See Adding Examples.

The above is an extract from [open-api docs](https://swagger.io/docs/specification/describing-parameters/#path-parameters)

RapiDoc too uses `example` or `examples`  to set values in the field

let me know if it clarifies your case ?

**Comment by nphmuller** [comment]
Thanks! That cleared it all up! I made some incorrect assumptions based on the behaviour of Swagger UI.

**Comment by mrin9** [comment]
this is now addressed in release `5.3.1` _Default value will be show if it exist_
